src/views/AmalgamationRegular/ReviewConfirm.vue

```diff
@@ -331,6 +331,17 @@ export default class AmalgamationRegularReviewConfirm extends Vue {
   @Action(useStore) setHasPlanOfArrangement!: (x: boolean) => void
   @Action(useStore) setIsFutureEffective!: (x: boolean) => void
 
+  /**
+   * In case submitting the amalgamation failed, we want to reset the validity of Certify.
+   * This is since the checkbox has to be ticked again after the save dialog has been closed.
+   */
+  mounted (): void {
+    this.setCertifyState({
+      certifiedBy: this.getCertifyState.certifiedBy,
+      valid: false
+    })
+  }
```

Comment:
Is this how other filings do it? Or do they need this, too?

Reply:
Good point. I think they need this too. The other filings don't do anything as of right now from what I checked.

Reply:
Should the Certify component report itself as invalid, so that this isn't needed?

Reply:
It can't do that I believe in this scenario. We're using the common one that's created from the shared components one.

The reason is, when the save error dialog is shown, at that point the component's validity is true. After it's closed, the UI keeps that validity. So, it's still true even though the checkbox is back to being unticked. It kept that from before.

Reply:
OK. Do it your way. I was hoping for something generic. Resolving.